bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZZTo2L2Q01j_5f_NLO_FXFX/ZZTo2L2Q01j_5f_NLO_FXFX_proc_card.dat

```diff
@@ -1,12 +1,9 @@
 import model loop_sm-ckm_no_b_mass
 define p = p b b~
 define j = j b b~
-
 define V = z
 define ell+ = e+ mu+ ta+
 define ell- = e- mu- ta-
-
 generate p p > V ell+ ell- [QCD] @0
 add process p p > V ell+ ell- j [QCD] @1
```

Comment:
This is also similar to the last comment I made in [1].
I don't think this is safe to use

[1] https://github.com/SMP-GEN-Coordination/mc-requests/issues/2

Reply:
@sihyunjeon
Okay. I will put ZZ one on a hold for now.